elfi/model/extensions.py

```diff
@@ -16,6 +23,7 @@ def __init__(self, name=None):
         """
         self._name = name or self.__class__.__name__
 
+    @classmethod
```

Comment:
Now this will be only usable with static classes. I would perhaps keep it as it was or make a separate definition for non static classes.